pom.xml

```diff
@@ -127,31 +190,18 @@
       <plugin>
         <groupId>org.jacoco</groupId>
         <artifactId>jacoco-maven-plugin</artifactId>
-        <version>${jacoco.version}</version>
-        <configuration>
-          <skip>${maven.test.skip}</skip>
-          <destFile>${basedir}/target/coverage-reports/jacoco-unit.exec</destFile>
-          <dataFile>${basedir}/target/coverage-reports/jacoco-unit.exec</dataFile>
-          <output>file</output>
-          <append>true</append>
-          <excludes>
-            <exclude>*MethodAccess</exclude>
-          </excludes>
-        </configuration>
         <executions>
           <execution>
-            <id>jacoco-initialize</id>
             <goals>
               <goal>prepare-agent</goal>
             </goals>
-            <phase>test-compile</phase>
           </execution>
           <execution>
-            <id>jacoco-site</id>
-            <phase>verify</phase>
+            <id>report</id>
             <goals>
               <goal>report</goal>
             </goals>
+            <phase>verify</phase>
```

Comment:
This is the [default](https://www.eclemma.org/jacoco/trunk/doc/report-mojo.html), can be omitted.

Reply:
right will change!

Reply:
https://github.com/corona-warn-app/cwa-verification-server/pull/57/commits/0eeb5bb84828afd77fed2e52ffe5d78d75ed3eef